modules/python_files/dummy_function.py

```diff
@@ -1,2 +1,2 @@
 def handler(event, context):
```

Comment:
What is this for? Test purposes only?

Reply:
I want to be able to deploy the infrastructure for a cronjob and that requires me to provide a lambda function, otherwise I cannot deploy all the infrastructure. I want this to be the only thing that is handled with terraform. The CI/CD pipeline and execution environment should be completely independent of the infrastructure code. The idea is taken from this article that you shared with me: https://johnroach.io/2020/09/04/deploying-lambda-functions-with-terraform-just-dont/